src/types/effects.ts

```diff
@@ -249,3 +257,70 @@ export type SignerSponsorshipRemoved = Omit<
   SignerSponsorshipEvents,
   "new_sponsor" | "sponsor"
 > & { type_i: EffectType.signer_sponsorship_removed };
+export interface LiquidityPoolEffectRecord extends Horizon.BaseResponse {
+  id: string;
+  fee_bp: number;
+  type: Horizon.LiquidityPoolType;
+  total_trustlines: string;
+  total_shares: string;
+  reserves: [
+    {
+      amount: string;
+      asset: string;
+    },
+  ];
```

Comment:
Seems like this is repeating in the types quite a bit (and I made a definition mistake earlier, as you saw). Maybe this should be a separate type, e.g.

```typescript
export interface Reserve {
    amount: string;
    asset: string;
}
```
and then
```suggestion
  reserves: Reserve[];
```

Reply:
Addressed in https://github.com/stellar/js-stellar-sdk/pull/690/commits/3588410d9d1c1b06fc727756334744bb6f3d8dbf.